src/SprykerEco/Yves/Unzer/Theme/default/credit_card.twig

```diff
@@ -0,0 +1 @@
+{% extends view('credit-card', 'Unzer') %}
```

Comment:
What is that? 
We can have only `template`, `view` or `component`.

Reply:
@kkicheglovspryker plz replace path to `credit_card` view and remove current twig file